libyara/grammar.y

```diff
@@ -181,6 +182,8 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 %token _ASCII_                                         "<ascii>"
 %token _WIDE_                                          "<wide>"
 %token _XOR_                                           "<xor>"
+%token _BASE64_                                        "<base64>"
+%token _BASE64_WIDE_                                   "<base64wide>"
```

Comment:
The keyword is actually `widebase64` but here it appears as `base64wide`. Actually I like `base64wide` more than `widebase64`, it's like the more relevant part of the modifier is at the beginning of the keyword and `wide` is just an addendum to `base64`.

Reply:
I had it as `base64wide` at some point and switched it for some reason I can no longer remember, but apparently not everywhere. I actually like `base64wide` better and I've now made it consistent everywhere.